html/screenreaders/demos/visually-hidden-text/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Визуально скрытый текст кнопки — Скринридеры — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    .body {
+      display: flex;
+      justify-content: center;
+      margin: 0;
+      padding: 50px;
+      background-color: #eee;
+      font-family: "Roboto", sans-serif;
+    }
+
+    .button {
+      border: 0;
+      background: transparent;
+      color: #424242;
+    }
+
+    .button__icon {
+      fill: currentColor;
+    }
+
+    .button__icon:hover {
+      fill: #e65100;
+    }
+
+    .button:focus {
+      outline: 3px solid #e65100;
+      outline-offset: 4px;
+    }
+
+    .visually-hidden {
+      position: absolute;
+      width: 1px;
+      height: 1px;
+      margin: -1px;
+      padding: 0;
+      border: 0;
+      clip: rect(0 0 0 0);
+      overflow: hidden;
+    }
+  </style>
+</head>
+<body class="body">
+  <button class="button">
```

Comment:
```suggestion
  <button>
```

В примере в статье у нас нет класса у кнопки.